### ISSUE TITLE
Fix discrepancy in handling key file version strings

### ISSUE THREAD

**Issue by TyKohler** [keepassium/KeePassium#143]
Using key files created via KeePass for Windows, KeePassium recently started throwing `KeyFileError.unsupportedFormat`. After further analysis, version "1.0" key files actually have a default version string of "1.00". Expected changes are to respect both potential version strings for version "1.0" key files. Version "2.0" key files do not appear to have the same variance.

Example:
```swift
internal override func processXmlKeyFile(keyFileData: ByteArray) throws -> SecureByteArray? {
    let xml = try AEXMLDocument(xml: keyFileData.asData)
    let version = xml[Xml2.keyFile][Xml2.meta][Xml2.version].value
    switch version {
    case "2.0":
        let result = try processXMLFileVersion2(xml) 
        return result
    case "1.0", "1.00": // Variance addressed here.
        let result = try processXMLFileVersion1(xml) 
        return result
    default:
        throw KeyFileError.unsupportedFormat
    }
}
```

1. Create or update a database with key file version "1.0" (I did this on Windows using KeePass).
2. Attempt to use both the database and the new key file with KeePassium.
3. You should see error "Unsupported key file format".

Platform specifics:
 - Device: iPhone 11 Pro Max
 - OS: iOS 14.4
 - App Version: 1.12.80 beta

**Comment by keepassium** [comment]
Yep, I am aware of the issue. It should be fixed in 1.21.81 beta (just pushed to TestFlight).

**Comment by TyKohler** [comment]
Confirmed to be fixed on 1.12.81 beta! Thanks!

**Comment by keepassium** [comment]
Thanks for confirming!
